src/library-authoring/list-libraries/data/api.js

```diff
@@ -49,6 +49,7 @@ export async function getLibraryList(params) {
         version: null,
         has_unpublished_changes: false,
         has_unpublished_deletes: false,
+        license: '',
```

Comment:
Noting a minor inconsistency: the default is set to `null` in LibraryEditPage.

Reply:
See the note above-- this is normalized to empty string, which is All Rights Reserved, since it's the default legal license and v1 libraries don't have a license field.

Reply:
Sounds good to me.  Thanks for the explanation!